core/controllers/classifier_test.py

```diff
@@ -197,10 +198,15 @@ def __init__(self) -> None:
         def mock_get_classifier_training_job_by_id(_: str) -> FakeTrainingJob:
             return FakeTrainingJob()
 
-        can_send_emails_ctx = self.swap(
-            feconf, 'CAN_SEND_EMAILS', True)
+        can_send_emails_ctx = (
+            self.swap_to_always_return(
```

Comment:
For this & other tests in this PR that use this pattern, might this have an unaffected side effect? Unless I'm mistaken, this will cause all boolean parameters to return true. Shouldn't we be configuring the tests such that only the parameter(s) we care about being true are changed as such?

Reply:
@kevintab95 Just a note, @lkbhitesh07 made a PR at https://github.com/oppia/oppia/pull/19756 to handle this more cleanly for feature flags. Maybe worth doing something similar for platform params?

Reply:
Done.